doc/guides/west/troubleshooting.rst

```diff
@@ -5,6 +5,36 @@ Troubleshooting West
 
 This page covers common issues with west and how to solve them.
 
+``west update`` fetching failures
+*********************************
+
+One good way to troubleshoot fetching issues is to run ``west update`` in
+verbose mode, like this:
+
+.. code-block:: shell
+
+   west -v update
+
+The output includes Git commands run by west and their outputs. Look for
+something like this:
+
+.. code-block:: none
+
+   === updating your_project (path/to/your/project):
+   west.manifest: your_project: checking if cloned
+   [...other west.manifest logs...]
+   --- your_project: fetching, need revision SOME_SHA
+   west.manifest: running 'git fetch ... https://github.com/your-username/your_project ...' in /some/directory
+
+The ``git fetch`` command example in the last line above is what needs to
+succeed. Go to ``/some/directory``, copy/paste and run the entire ``git fetch``
+command, then debug from there using the documentation for your credential
+storage helper.
```

Comment:
git has most often proved useless to debug proxy, authentication or other network issues. It's generally required to peel yet another layer of indirection and use `curl -v` or `ssh -v`. Maybe mention that briefly.

Reply:
> `curl -v` or `ssh -v`. Maybe mention that briefly.

Easy enough and sounds wise. Added a commit.